assets/src/components/v2/screen_container.tsx

```diff
@@ -71,10 +83,15 @@ const ScreenLayout: ComponentType<ScreenLayoutProps> = ({
 
 const ScreenContainer = ({ id }) => {
   const blinkConfig = useContext(BlinkConfigContext);
+  const audioConfig = useContext(AudioConfigContext);
   const [showBlink, setShowBlink] = useState(false);
 
   const { apiResponse, requestCount } = useApiResponse({ id });
 
+  if (audioConfig != null) {
```

Comment:
Ideally you don't want to conditionally run hooks within a component since it can alter the order of hooks are called in subsequent renders. https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

You could pass the full `audioConfig` object to the hook itself and do the null check there and return early if the config is null. 

Additional question on this line, should this be `!==` null since this is checking for the default value which is `null`